src/source/sine.rs

```diff
@@ -59,11 +57,7 @@ impl Source for SineWave {
     }
 
     #[inline]
-    fn try_seek(&mut self, _: Duration) -> Result<(), SeekError> {
-        // This is a constant sound, normal seeking would not have any effect.
-        // While changing the phase of the sine wave could change how it sounds in
-        // combination with another sound (beating) such precision is not the intend
-        // of seeking
-        Ok(())
```

Comment:
see the thread on try_seek. I would propose make the original comment a doc comment and mentioning `skip_samples` as the solution for things like beating

Reply:
In [322bdd5](https://github.com/RustAudio/rodio/pull/602/commits/322bdd5f5f66d1930f09e193fc1f48314ec11626)